conversation/src/main/java/com/ibm/watson/developer_cloud/conversation/v1/model/message/MessageResponse.java

```diff
@@ -30,8 +30,8 @@
   private static final String TEXT = "text";
 
   private Map<String, Object> context;
-  private List<Entity> entities;
-  private List<Intent> intents;
+  private List<MessageEntity> entities;
```

Comment:
This is a breaking change, can we keep `Entity` and find a new name for `MessageEntity` is the object that different from `Entity` ?